.gitignore

```diff
@@ -121,8 +121,7 @@ thirdparties/rocksdb/rocksdb/
 thirdparties/rocksdb/*log
 thirdparties/rocksdb/*.tar.gz
 thirdparties/aws/*.tar.gz
-thirdparties/etcdclient/tmp/
-thirdparties/etcdclient/*.h
+thirdparties/etcdclient/*
```

Comment:
Modifications to files other than `tmp`  under `thirdparties/etcdclient` cannot be ignored.

Reply:
We usually do not change the contents of this folder, instead when compiling the project, a lot of extra files are created in this directory.

Reply:
> Modifications to files other than `tmp` under `thirdparties/etcdclient` cannot be ignored.
![2022-12-02_12-12-1669954772](https://user-images.githubusercontent.com/75058860/205214013-595a98cf-f5c0-4071-b916-5ebfd5b69c7d.jpg)
![2022-12-01_15-12-1669878656](https://user-images.githubusercontent.com/75058860/205214176-fee49dd4-1f15-4f14-9fc2-2f5bf9adf577.jpg)


here you can see it's inconvenient here. And someone use vscode git plugin. so ugly here.